src/shared/components/iframe-runtime.tsx

```diff
@@ -117,9 +121,24 @@ export const IframeRuntime: React.FC<IProps> =
         phoneRef.current.disconnect();
       }
     };
-  },[addLocalLinkedDataListener, authoredState, logRequestData, report, setHint, url, initMessage]);
+  },[addLocalLinkedDataListener, authoredState, logRequestData, report, setHint, url, initMessage, view]);
 
-  const iframeStyle = iframeStyling ?? {width: "100%", height: iframeHeight, border: "none"};
+  let scaledIframeStyle = undefined;
+  if (scale && report && view !== "standalone") {
+    scaledIframeStyle = {
+      border: "none",
+      height: `${300/scale}px`,
```

Comment:
It would be better if `iframeHeight` was used here instead of 300. This way interactives that use the interactive api to request more space will still work properly.